eclipse-collections-api/pom.xml

```diff
@@ -88,7 +88,7 @@
                     <windowtitle>Eclipse Collections API - ${project.version}</windowtitle>
                     <show>public</show>
                     <links>
-                        <link>https://docs.oracle.com/javase/8/docs/api/</link>
+                        <link>https://docs.oracle.com/en/java/javase/17/docs/api</link>
```

Comment:
Why was this changed from 8 to 17 instead of 8 to 11? Was this because we are building Javadoc with 17?

Reply:
> Why was this changed from 8 to 17 instead of 8 to 11? Was this because we are building Javadoc with 17?

Never mind @prathasirisha, I see you added a comment above referencing this change.

Reply:
I meant to reference Java 11 docs, looks like committed the changes before I could make that change. I will update them shortly.